test_cases/september_test_case.json

```diff
@@ -80,6 +89,15 @@
                 {
                     "@type": "ElectionResults.CandidateContest",
                     "@id": "recsoZy7vYhS3lbcK",
+                    "ExternalIdentifier": [
```

Comment:
This `ExternalIdentifier` is materially different from the one in the [August EDF](https://github.com/TrustTheVote-Project/NIST-1500-100-103-examples/blob/main/test_cases/august_test_case.json#L84-L91), which looks like this:

```json
                    "ExternalIdentifier": [
                        {
                            "@type": "ElectionResults.ExternalIdentifier",
                            "Type": "other",
                            "OtherType": "https://viaf.org/viaf/",
                            "Value": "129529146"
                        }
                    ],
```

Markit's logic is built to that spec, and specifically looks for an `ExternalIdentifier` whose `Value` is `129529146`. It therefore fails to parse this EDF, as it doesn't see this as a presidential contest.

Which is correct?

Reply:
@mattbeattie this is up to you; these are not real data. I'm not sure what the "right" answer is: if the ExternaIIdentifier is a URI, then its value should be `https://viaf.org/viaf/129529146`; the strategy of splitting the context (`https://viaf.org/viaf`) from the ID (`129529146`) is commonly used -- I've used it myself -- to shorten IDs, but I think it's not a good idea in general.  But again: this is your data!

Reply:
I agree, I like what you have here better. Let's use it. Everything's good to go on Markit!